docs/content/intro/install.md

```diff
@@ -99,26 +99,26 @@ $env:PATH+=";$env:USERPROFILE\bin"
    
     If the deployment fails, proceed to the [troubleshooting](#troubleshooting) section.
 
+## Install Brigade Gateways
+
 Now that you have the Brigade server installed, an optional next step is to
 install a Brigade [Gateway]. All of the current gateways rely on remote access
 to Brigade's API server, so a Kubernetes cluster that can provision public
 IP addresses will be necessary for gateway functionality. (Alternatively,
 another means for forwarding traffic to the API server may be used, such as
 [ngrok](https://ngrok.com/).)
 
-## Install Brigade Gateways
-
-A Brigade [Gateway] generates events in Brigade in response to external triggers, for example a git push to repository.
-Brigade does not install gateways by default, which limits inbound events to those generated by the brig CLI,
-such as `brigade.sh/cli:exec` (the event source and type generated from the `brig event create` command.)
-
-Currently, the list of official Brigade v2 gateways that process external events is as follows:
+A Brigade [Gateway] generates events in Brigade in response to external
+triggers, for example a git push to repository. Brigade does not install
+gateways by default, which limits inbound events to those generated by the brig
```

Comment:
This wording could trip up people who were used to gateways being included in Brigade 1.x that just needed to be enabled.

Could we reword to something like "Brigade does not ship with any gateways, which limits......... but compatible gateways are easy to install as add-ons" ?